Headers/state_diag_mod.F90

```diff
@@ -10581,6 +10701,11 @@ SUBROUTINE Get_Metadata_State_Diag( am_I_Root,  metadataID, Found,         &
        IF ( isUnits   ) Units = 's-1'
        IF ( isRank    ) Rank  = 3
 
+    ELSE IF ( TRIM( Name_AllCaps ) == 'NOXTAU' ) THEN
```

Comment:
Do you want this diagnostic added to the standard model?

Reply:
Based on other comment I am separating the NOxTau diagnostic into MODEL_GEOS blocks.